components/carousel/index.en-US.md

```diff
@@ -18,6 +18,8 @@ A carousel component. Scales with its container.
 | --- | --- | --- | --- | --- |
 | afterChange | Callback function called after the current index changes | function(current) | - |  |
 | autoplay | Whether to scroll automatically | boolean | `false` |  |
+| autoplaySpeed | The interval of carousel | number | 3000 |  |  
```

Comment:
I don't think we should add this, `Carousel` base on [react-slick](https://github.com/akiran/react-slick)